src/main/java/seedu/commands/QuizCommand.java

```diff
@@ -17,8 +17,16 @@
 
     private int subjectIndex;
 
+    private int numToQuiz;
+
     public QuizCommand(int subjectIndex) {
         this.subjectIndex = subjectIndex - 1;
+        this.numToQuiz = -1;
+    }
+
+    public QuizCommand(int subjectIndex, int numToQuiz) {
+        this.subjectIndex = subjectIndex - 1;
+        this.numToQuiz = numToQuiz;
```

Comment:
You might want to check and ensure that numToQuiz is not greater than the number of cards!